definitions/ext-rf_scanner/summary_metrics.yml

```diff
@@ -14,20 +14,21 @@ volume:
   query:
     select: count(*)
     from: THD_RF_TRANSACTION
-    eventId: entity.guid
 
-errors:
-  title: Errors
-  unit: COUNT
-  query:
-    select: count(message)
-    from: Log
-    where: message like '%error%' or message like '%Error%' and message NOT LIKE '%lqqqq%' or message NOT LIKE '%Winsock error 10042%'
     eventId: entity.guid
+# Comment out as Log eventtype does not have a entity.guid
+#errors:
+#  title: Errors
+#  unit: COUNT
+#  query:
+#    select: count(message)
+#    from: Log
+#    where: message like '%error%' or message like '%Error%' and message NOT LIKE '%lqqqq%' or message NOT LIKE '%Winsock error 10042%'
+#    eventId: entity.guid
     
 lastactive:
   title: Last Active Time
-  unit: STRING
+  unit: TIMESTAMP
```

Comment:
Since you removed this tag in the definition. You will need to retrieve this from some metric.

I understand this will be something like `SELECT latest(lastActiveTimestamp) FROM THD_RF_TRANSACTION` ?